### ISSUE TITLE
Future enhancement

### ISSUE THREAD

**Issue by FDewaleyne** [FDewaleyne/regen-repodata#3]
new version with tracking of the status for all the channels that have to be regenerated (api only)

**Comment by FDewaleyne** [comment]
tracking the status should be left to another script running in parallel - the reason being based in how the pulling of the new jobs is done in satellite 5 (you need to both have taskomatic running and the channel repodata job queued correctly in the satellite). api calls are also taken into account far more quickly than the db insertions.

**Comment by FDewaleyne** [comment]
solving through another issue
